index.html

```diff
@@ -41,22 +41,20 @@ <h1>Get ready to Spot the Surveillance...</h1>
         <img id="sts-sky" src="assets/images/sky-lighten.jpg">
 
         <!-- sounds -->
-        <audio id="alpr-audio" src="assets/audio/alpr.mp3"></audio>
-        <audio id="alpr-mobile-audio" src="assets/audio/alpr-mobile.mp3"></audio>
-        <audio id="biometric-audio" src="assets/audio/biometric.mp3"></audio>
-        <audio id="bodycam-audio" src="assets/audio/bodycam.mp3"></audio>
-        <audio id="camera-audio" src="assets/audio/camera.mp3"></audio>
-        <audio id="camera2-audio" src="assets/audio/camera2.mp3"></audio>
-        <audio id="drone-audio" src="assets/audio/drone.mp3"></audio>
-        <audio id="bird-audio" src="assets/audio/bird.mp3"></audio>
-        <audio id="light-audio" src="assets/audio/light.mp3"></audio>
+        <audio id="alpr-audio" src="assets/audio/alpr.ogg"></audio>
+        <audio id="alpr-mobile-audio" src="assets/audio/alpr-mobile.ogg"></audio>
+        <audio id="biometric-audio" src="assets/audio/biometric.ogg"></audio>
+        <audio id="bodycam-audio" src="assets/audio/bodycam.ogg"></audio>
+        <audio id="camera-audio" src="assets/audio/camera.ogg"></audio>
+        <audio id="camera2-audio" src="assets/audio/camera2.ogg"></audio>
+        <audio id="drone-audio" src="assets/audio/drone.ogg"></audio>
+        <audio id="bird-audio" src="assets/audio/bird.ogg"></audio>
+        <audio id="light-audio" src="assets/audio/light.ogg"></audio>
         <audio id="intro-audio" src="assets/audio/intro-delayed.mp3"></audio>
-        <audio id="exit-audio" src="assets/audio/exit.mp3"></audio>
-        <audio id="congrats-audio" src="assets/audio/congrats.mp3"></audio>
-        <audio id="beep"  src="assets/audio/beeps.mp3"></audio>
-        <audio id="boop"  src="assets/audio/boop.wav"></audio>
-        <audio id="ambience" src="assets/audio/traffic-fade.mp3"></audio>
-        <audio id="ding"  src="assets/audio/piano-ding-eff.mp3"></audio>
+        <audio id="exit-audio" src="assets/audio/exit.ogg"></audio>
+        <audio id="congrats-audio" src="assets/audio/congrats.ogg"></audio>
+        <audio id="ambience" src="assets/audio/traffic-fade.ogg"></audio>
+        <audio id="ding"  src="assets/audio/piano-ding-eff.ogg"></audio>
 
         <!-- mixins -->
 
```

Comment:
Yup, this will be rebased against master with https://github.com/EFForg/spot_the_surveillance/commit/7333872000b0c57584083f132732f71cc421b82a that changes the camera cards.

Reply:
ah, that's the commit. thanks. the rebase didn't apply correctly for me, or I'm reading it wrong.

Reply:
These are different image and audio files, so you'll just have to swap them out and it should just work!